projects/samples/devices/controllers/hokuyo/hokuyo.c

```diff
@@ -68,9 +65,11 @@ static void display(WbDeviceTag d,   // display
 
   int i;
   for (i = 0; i < ns; i++) {
-    float f = MIN(CLAMP_MAX, v[i]);
-    float alpha = -fov2 + fov * i / ns - M_PI_2;
-    wb_display_draw_line(d, dw2, dh2, dw2 + f * cos(alpha) * dw2 / CLAMP_MAX, dh2 + f * sin(alpha) * dh2 / CLAMP_MAX);
+    const float f = v[i];
+    if (f != INFINITY) {
+      float alpha = -fov2 + fov * i / ns - M_PI_2;
```

Comment:
Maybe `alpha` could also be set const
```suggestion
      const float alpha = -fov2 + fov * i / ns - M_PI_2;
```